pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UserAdminService.java

```diff
@@ -637,14 +630,14 @@ protected static int verifyStudyGroup(String studyGroup) {
     protected Map<String, RoleInfo> verifyOperatorAdminRoles(int operatorId, int groupId) {
         Map<String, RoleInfo> adminRoles = getOperatorAdminRoles(operatorId, groupId);
         if (adminRoles == null || adminRoles.isEmpty()) {
-            throw new DSMBadRequestException("Operator does not have user administrator privileges for study group "
+            throw new DSMBadRequestException("Operator " + operatorId + " does not have user administrator privileges for study group "
                     + studyGroup);
         }
         return adminRoles;
     }
 
     protected static Map<String, RoleInfo> getOperatorAdminRoles(int operatorId, int groupId) {
-        Map<String, RoleInfo> adminRoles = null;
+        Map<String, RoleInfo> adminRoles = Collections.emptyMap();
```

Comment:
Not sure why this was set to null--when I was building out the test, I hit an NPE here so I changed it to an empty map by default.